app/models/financial_aid/my_financial_aid_summary.rb

```diff
@@ -28,12 +30,15 @@ def aid
         aid[aid_year_id.to_s] = {
           totalCostOfAttendance: format_currency(financial_aid_summary.try(:[], 'uc_cost_attendance')),
           totalGiftAidAndWaivers: format_currency(financial_aid_summary.try(:[], 'uc_gift_aid_waiver')),
+          totalThirdPartyContracts: format_currency(financial_aid_summary.try(:[], 'uc_third_party')),
           totalNetCost: format_currency(financial_aid_summary.try(:[], 'uc_net_cost')),
           totalFundingOffered: format_currency(financial_aid_summary.try(:[], 'uc_funding_offered')),
           giftAidAndOutsideResources: format_currency(financial_aid_summary.try(:[], 'uc_gift_aid_out')),
           grantsAndScholarships: format_currency(financial_aid_summary.try(:[], 'uc_grants_schol')),
+          outsideResources: format_currency(financial_aid_summary.try(:[], 'uc_outside_resrces')),
           waiversAndOtherFunding: format_currency(financial_aid_summary.try(:[], 'uc_waivers_oth')),
           feeWaivers: format_currency(financial_aid_summary.try(:[], 'uc_fee_waivers')),
+          thirdPartyContracts: format_currency(financial_aid_summary.try(:[], 'uc_third_party')),
```

Comment:
@marcpechaitis - Why are we including this same value again? We've got it above as `totalThirdPartyContracts` already.

Reply:
@redconfetti I was trying to be consisted with the existing API for the other items on the page / allow room for "what if some day there is another item that would be included in `totalThirdPartyContracts`"?  Its easy to change... thoughts?

Reply:
@marcpechaitis - Sounds good